Lib/venv/__init__.py

```diff
@@ -303,8 +303,11 @@ def setup_python(self, context):
             copier(context.executable, path)
             if not os.path.islink(path):
                 os.chmod(path, 0o755)
-            for suffix in ('python', 'python3',
-                           f'python3.{sys.version_info[1]}'):
+
+            suffixes = ['python', 'python3', f'python3.{sys.version_info[1]}']
+            if sys.version_info[:2] == (3, 14):
```

Comment:
Do we need this `if`? It's always true in this branch, and it'll be removed in the next.

Reply:
There was some concern on the issue that we would forget to remove it, or that needing to remember to remove would be a maintenance burden.

Happy to go with what you feel is best!

Reply:
What we could do is raise an error in 3.15?

```python
assert sys.version_info[:2] == (3, 14), "python version is not pi"
```

Reply:
I thought of that, but if we're making a version check, might as well be the existing one.